memory/memory.ml

```diff
@@ -14,190 +14,224 @@
-let pages_of_kib_used   value = divide_rounding_up value kib_per_page
-let mib_of_bytes_used   value = divide_rounding_up value bytes_per_mib
-let mib_of_kib_used     value = divide_rounding_up value kib_per_mib
-let mib_of_pages_used   value = divide_rounding_up value pages_per_mib
-
-(* === Domain memory breakdown ======================================================= *)
-
-(*           ╤  ╔══════════╗                                              ╤            *)
-(*           │  ║ shadow   ║                                              │            *)
-(*           │  ╠══════════╣                                              │            *)
-(*  overhead │  ║ extra    ║                                              │            *)
-(*           │  ║ external ║                                              │            *)
-(*           │  ╠══════════╣                                   ╤          │            *)
-(*           │  ║ extra    ║                                   │          │            *)
-(*           │  ║ internal ║                                   │          │            *)
-(*           │  ╠══════════╣  ╤    ╤                 ╤         │          │ footprint  *)
-(*           │  ║ shim     ║  │    │                 │         │          │            *)
-(*           ╪  ╠══════════╣  ╧    ╧        ╤        │         │ xen      │            *)
-(*           │  ║ video    ║                │        │ actual  │ maximum  │            *)
-(*           │  ╠══════════╣  ╤    ╤        │        │ /       │          │            *)
-(*           │  ║          ║  │    │ build  │ target │ total   │          │            *)
-(*           │  ║ guest    ║  │    │ start  │        │         │          │            *)
-(*    static │  ║          ║  │    │        │        │         │          │            *)
-(*   maximum │  ╟──────────╢  │    ╧        ╧        ╧         ╧          ╧            *)
-(*           │  ║          ║  │                                                        *)
-(*           │  ║          ║  │                                                        *)
-(*           │  ║ balloon  ║  │ build                                                  *)
-(*           │  ║          ║  │ maximum                                                *)
-(*           │  ║          ║  │                                                        *)
-(*           ╧  ╚══════════╝  ╧                                                        *)
-
-(* === Domain memory breakdown: HVM guests =========================================== *)
+
+let pages_of_kib_used value = divide_rounding_up value kib_per_page
+
+let mib_of_bytes_used value = divide_rounding_up value bytes_per_mib
+
+let mib_of_kib_used value = divide_rounding_up value kib_per_mib
+
+let mib_of_pages_used value = divide_rounding_up value pages_per_mib
+
+(* === Domain memory breakdown
+   ======================================================= *)
+
+(* ╤ ╔══════════╗ ╤ *)
```

Comment:
:(

Reply:
Oops, I think we can skip the formatting for this comment

Reply:
This got fixed by adding an ocaml attribute to the comment. It still means these kind of comments can get broken by enabling wrapping of comments and probably why it's not enabled by default.

The problem is that if it's not enabled all the poorly-formatted comments do not get fixed. :/

Reply:
it's a shame about the comments, but we're doing the reformat for the code, so i think it's a reasonable sacrifice

Reply:
I'd rather have this unresolved so it's visible as a possible issue

Reply:
Have you tried to raise an issue on the ocamlformat repo? They are usually quite helpful and responsive

Reply:
I'm not sure what to report. This is the expected behaviour when turning wrapping on for comments, it's supposed to treat the comments as paragraphs, which breaks pretty diagrams :)

Reply:
Well, one could ask for a way to disable wrapping locally for selected comments, which I think can be useful in many cases

Reply:
That's in their readme: `[@@ocamlformat "disable"]`

It's being used for this comment ;)

Reply:
🤣 sorry